packages/myst-frontmatter/package.json

```diff
@@ -36,10 +36,12 @@
     "doi-utils": "^2.0.0",
     "orcid": "^1.0.0",
     "simple-validators": "^1.0.4",
-    "spdx-correct": "^3.2.0"
+    "spdx-correct": "^3.2.0",
+    "myst-toc": "^0.1.0"
   },
   "devDependencies": {
     "@types/spdx-correct": "^3.1.0",
+    "@vitest/coverage-v8": "^1.6.0",
```

Comment:
I don't think this is used at the moment? If so should come in at the top-package where vitest, typescript etc come in.

Reply:
Yes, my bad. Unrelated work sneaking in again.